geth/txqueue/transactions.go

```diff
@@ -0,0 +1,59 @@
+package txqueue
+
+import (
+	"sync"
+
+	"github.com/status-im/status-go/geth/common"
+)
+
+// transactions safely holds queued transactions.
+type transactions struct {
+	m map[common.QueuedTxID]*common.QueuedTx
+	l sync.RWMutex
```

Comment:
Mutex as `mu` and normally often as first field.

Reply:
ok